15LessonFifteen/Checkpoint3/css/css-reset.css

```diff
@@ -0,0 +1,20 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
don't include a reset if you're using bootstrap, it might interfere with bootstrap!